lib/ruby_indexer/lib/ruby_indexer/declaration_listener.rb

```diff
@@ -17,10 +17,11 @@ class DeclarationListener
         dispatcher: Prism::Dispatcher,
         parse_result: Prism::ParseResult,
         file_path: String,
+        collect_documentation: T::Boolean,
         enhancements: T::Array[Enhancement],
       ).void
     end
-    def initialize(index, dispatcher, parse_result, file_path, enhancements: [])
+    def initialize(index, dispatcher, parse_result, file_path, collect_documentation: false, enhancements: [])
```

Comment:
Since we ultimately aim to extract the indexer to its own gem, I think it would be less-surprising if we default to `true` for `collect_documentation`.

Reply:
Great point.